Automattic/pocket-casts-android
# End of Year: Add stories structure

| 📘 Project: #410 |
|:---:|

Adds the initial stories screen. No functionality is built on this PR, just the view.

iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/383

https://user-images.githubusercontent.com/1405144/196947652-623294c5-9dec-4992-b18a-b390026c7bb5.mp4



## To test

1. Set `END_OF_YEAR_ENABLED` feature flag to `true` in `base.gradle`
2. Run the app
3. When the prompt appears, tap "View My 2022"
4. ✅ Check that the stories view appear
5. Tap close
6. ✅ Check that the stories view is dismissed

# Checklist

- Should this change be included in the release notes? If yes, please add a line in CHANGELOG.md
- [ ] Have you tested in landscape? - iOS restricts the screen to portrait mode, this will be taken up in a future PR
- Does the change work with a large display font?
- Are all the strings localized? 
- Could you have written any new tests?
